google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -339,8 +363,7 @@ public void testJsonStreamWriterWithDefaultStream()
     bigquery.create(tableInfo);
     TableName parent = TableName.of(ServiceOptions.getDefaultProjectId(), DATASET, tableName);
     try (JsonStreamWriter jsonStreamWriter =
-        JsonStreamWriter.newBuilder(parent.toString(), tableInfo.getDefinition().getSchema())
```

Comment:
This changes public interface, could it break users?

Reply:
yes it would break preview users